ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.17.yaml

```diff
@@ -413,6 +347,190 @@ tests:
+    - ref: openshift-qe-cluster-density-v2
+  timeout: 8h0m0s
+- always_run: false
+  as: perfscale-aws-ovn-medium-node-density-cni
+  optional: true
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale
+    env:
+      BASE_DOMAIN: perfscale.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m5.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m5.8xlarge
+      EXTRA_FLAGS: --churn-duration=20m --service-latency
+      PROFILE_TYPE: both
+      WORKER_REPLICA_COUNT: "120"
+      ZONES_COUNT: "3"
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
+    - chain: workers-scale
+    - chain: create-infra-move-ingress-monitoring-registry
+    - ref: openshift-qe-node-density-cni
+  timeout: 8h0m0s
+- always_run: false
+  as: ovncore-perfscale-aws-ovn-large-cluster-density-v2
+  optional: true
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-ovn-perfscale
+    env:
+      BASE_DOMAIN: core.ovn.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m5.2xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: r5.8xlarge
+      EXTRA_FLAGS: --churn-duration=20m --service-latency --check-health=false
+      ITERATION_MULTIPLIER_ENV: "6"
+      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: r5.8xlarge
+      PROFILE_TYPE: both
+      SET_ENV_BY_PLATFORM: custom
+      WORKER_REPLICA_COUNT: "501"
+      ZONES_COUNT: "3"
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
+    - chain: workers-scale
+    - chain: create-infra-move-ingress-monitoring-registry
+    - ref: openshift-qe-cluster-density-v2
+  timeout: 8h0m0s
+- always_run: false
+  as: ovncore-perfscale-aws-ovn-large-node-density-cni
+  optional: true
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-ovn-perfscale
+    env:
+      BASE_DOMAIN: core.ovn.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m5.2xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: r5.8xlarge
+      EXTRA_FLAGS: --churn-duration=20m --service-latency --check-health=false
+      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: r5.8xlarge
+      PROFILE_TYPE: both
+      SET_ENV_BY_PLATFORM: custom
+      WORKER_REPLICA_COUNT: "501"
+      ZONES_COUNT: "3"
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
+    - chain: workers-scale
+    - chain: create-infra-move-ingress-monitoring-registry
+    - chain: openshift-qe-node-density-cni
+  timeout: 8h0m0s
+- always_run: false
+  as: ovncore-perfscale-aws-ovn-xlarge-cluster-density-v2
+  optional: true
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-ovn-perfscale
+    env:
+      BASE_DOMAIN: core.ovn.devcluster.openshift.com
+      CLUSTER_NETWORK_CIDR: 20.0.0.0/8
+      COMPUTE_NODE_TYPE: m5.2xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: r5.8xlarge
+      EXTRA_FLAGS: --churn-duration=20m --service-latency --check-health=false
+      ITERATION_MULTIPLIER_ENV: "6"
+      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: r5.8xlarge
+      PROFILE_TYPE: both
```

Comment:
Give the amount of nodes I recommend you o use the reporting profile to prevent generating thousands of docs

Reply:
Updated xlarge/larges job to have reporting profile.